app/src/main/java/com/rentminder/MainActivity.kt

```diff
@@ -55,34 +56,20 @@ class MainActivity : ComponentActivity() {
 
 @Composable
 fun MainMenu() {
-    //Getting current month for Main Menu
-    val cal: Calendar = Calendar.getInstance()
-    val monthDate = SimpleDateFormat("MMMM")
-    val monthName: String = monthDate.format(cal.time)
 
-    Column() {
-        TopToolBar()
-        Row(modifier = Modifier
-            .padding(
-                top = 5.dp,
-                bottom = 10.dp
-            )
-            .fillMaxWidth(),
-            horizontalArrangement = Arrangement.Center
-        ) {
-            Text(
-                text = monthName,
-                fontSize = 28.sp,
-                fontWeight = FontWeight.Bold
-            )
-        }
-        EditBillAmounts()
+    Column {
+        TopNavBarColumn()
+        CalendarNameColumn()
+        RentBillColumn()
```

Comment:
I changed the main menu column to be broken down into 3 different sub columns.

This follows the good practice in Kotlin for developers to read logic from top to bottom to understand what is happening. https://kotlinlang.org/docs/coding-conventions.html#interface-implementation-layout